Makefile

```diff
@@ -152,6 +152,14 @@ generate-timezones:
 generate-rtl:
 	go run ./scripts/characterorder/main.go | gofmt > pkg/util/intl/rtl_map.go
 
+.PHONT: generate-material-icons
+generate-material-icons:
+	pushd ./scripts/python && pip install -r requirements.txt && python ./subset_material_icons.py && popd
```

Comment:
Should use venv. Otherwise running this make target will pollute the running machine.